Add Hindi Translation

midroid opened the pull request:
Hi @reustle 

Creating a PR for Hindi Language Support. Felt it might be helpful for people who need information in Hindi/Nepali/Sanskrit/Marathi or other Indic languages because it shares the similar script for writing.

File Added: src/i18n/hi.json

File Updated: src/i18n/index.js

Also, Please find the attached screenshots for the UI/Visual changes.


![Screen Shot 2020-05-22 at 18 50 41](https://user-images.githubusercontent.com/3123099/82655538-4adb5080-9c5d-11ea-942b-285082ea09e4.png)
![Screen Shot 2020-05-22 at 18 50 58](https://user-images.githubusercontent.com/3123099/82655552-4dd64100-9c5d-11ea-9789-02abe52a08c4.png)
![Screen Shot 2020-05-22 at 18 51 12](https://user-images.githubusercontent.com/3123099/82655555-4e6ed780-9c5d-11ea-81ff-b0d494e292f0.png)

liquidx commented:
Thank you for your pull request. We merged in a new language earlier, would you be able to resolve the conflicts so I could merge this in? 

Thanks!

midroid commented:
Hi @liquidx , 

By mistake, I deleted the current PR branch tree track(repository basically), so I have to made a new PR at https://github.com/reustle/covid19japan/pull/349. 

As suggested, I have fixed the conflicts. But also found a new bug regarding introduction of new countries iso translations module. Please find the details there.

You can close this PR without merging and remove the branch.
